packages/payload/src/uploads/imageResizer.ts

```diff
@@ -312,23 +312,21 @@ export async function resizeAndTransformImageSizes({
         if (!resizeHeight) resizeHeight = resizeImageMeta.height
         if (!resizeWidth) resizeWidth = resizeImageMeta.width
 
-        // if requested image is larger than the incoming size, then resize using sharp and then extract with focal point
-        if (resizeHeight > resizeImageMeta.height || resizeWidth > resizeImageMeta.width) {
-          const resizeAspectRatio = resizeWidth / resizeHeight
-          const prioritizeHeight = resizeAspectRatio < originalAspectRatio
-          resized = imageToResize.resize({
-            height: prioritizeHeight ? resizeHeight : undefined,
-            width: prioritizeHeight ? undefined : resizeWidth,
-          })
-
-          // must read from buffer, resize.metadata will return the original image metadata
-          const { info } = await resized.toBuffer({ resolveWithObject: true })
-          resizeImageMeta.height = extractHeightFromImage({
-            ...originalImageMeta,
-            height: info.height,
-          })
-          resizeImageMeta.width = info.width
-        }
+        // Resize the image using sharp and only then extract with focal point
```

Comment:
I would adjust this to read: "scales the image before extracting from it"

I would place this comment above the `resized = imageToResize.resize({` section